pre_commit/resources/hook-tmpl

```diff
@@ -8,6 +8,7 @@ popd > /dev/null
 
```

Comment:
The file says it is a random(ish) md5. How is it created? How should I update it?